mountpoint-s3/src/cli.rs

```diff
@@ -888,6 +889,7 @@ where
                 client.clone(),
                 &args.bucket_name,
                 args.cache_block_size_in_bytes(),
+                EXPRESS_CACHE_MAX_OBJECT_SIZE,
```

Comment:
Time to add a `Config` type for the express cache? We could set a default value for max_object_size there.

Reply:
I can add the following struct to group configuration arguments of `ExpressDataCache::new`:
```rust
/// Configuration for a [ExpressDataCache].
#[derive(Debug)]
pub struct ExpressDataCacheConfig {
    /// Name of a S3 Express bucket to store the blocks.
    bucket_name: String,
    /// Name of the mounted bucket.
    source_bucket_name: String,
    /// Size of data blocks.
    block_size: u64,
    /// The maximum size of an object to be cached.
    max_object_size: usize,
}
```

I wouldn't implement a `Default` for it though, because only `max_object_size` has a sensible default value. I'd just leave `EXPRESS_CACHE_MAX_OBJECT_SIZE` constant for that purpose.